Analysis/MainScript.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+from ToothFractureRoutines import *
+
+'''
+This script runs Tooth fracture simulation and analysis on a set of directories given the plane equations at which the
+ fractures are to be simulated
+
+Steps for running the full pipeline:
+* Prep the data for each case:
+---- ToothCBCT.nnrd: has the grayscale image
+---- ToothCBCT-label.nrrd: is the labelmap image with segmentation. Label 1: tooth, Label 2: is a set of grayscale values
+                            that the fracture will be filled with after forming a distribution.
+---- ToothCBCT-resampled.nrrd: is the resampled version of original grayscale. Will be used to do wavelet analysis on.
+
+NOTE:
+To generate resampled grayscale, you can simulate fracture first, which will give a resampled grid (grid_*.nrrd),
+and use that as a reference volume in Slicer's 'Resample Scalar/Vector/DWI Volume', and ToothCBCT as input volume.
+
+* Get ALL the dependencies as listed ToothFractureRoutines.py and ToothFractureROCAnalysis.py
+* Setup the ToothFractureStatistics Slicer Extension.
+* Set the parameters in this script AND adjust paths to tools in ToothFractureRoutines.py
+* If you have setup Python virtual environment, start now
+* Run this script once with simulateFractures True, and once with simulateFractures False
+    Outputs: Wavelets and fracture simulated grayscale images
+* Run the SlicerExtension: ToothFractureStatistics
+    Input: The data directory (dataDirectory parameter below) with all the wavelets and grayscales calculated
+    Output: A csv file
+* Deactivate python virtual environment if started
+* Run the ToothFractureROCAnalysis.py
+    Input: CSV file generated in the previous step
+    Output: ROC curve analysis plots.
+
+'''
+
+
+# Set directory names and plane equations for fracture simulation
+# plane equation obtained from slicer using easyclip over the input image and target tooth
+DataSetDict = {}
+DataSetDict['CBCT_FB_2ndMolar_left'] = [ [ 0.0 , 0.0 , 1.0 , -21.7779791599 ],
+                   [0.960189391059, 0.0447772320061, 0.275737797177, 20.7913901406],
+                   [0.881835380307, 0.457810084008, 0.113032247695, 38.1997245286],
+                   [-0.475218084857, 0.696086834128, -0.538173662658, 26.1265530322],
+                   [0.205808735441, -0.823558891086, -0.528576879299, -15.2868925338],
+                   [0.935508951822, -0.205383765667, -0.287472624544, 22.2754210759] ]
+# DataSetDict['CBCT_FB_2ndMolar_right'] = [
```

Comment:
Why commenting these lines out? It might be cleaner to leave this uncommented and let the user change only the parameters that they really need to change.